Dockerfile-linux

```diff
@@ -43,15 +41,12 @@ RUN \
 RUN \  
   curl -fsSL "https://github.com/libevent/libevent/releases/download/release-$libevent_version/libevent-$libevent_version.tar.gz" -o libevent-$libevent_version.tar.gz && \
   curl -fsSL "https://github.com/libevent/libevent/releases/download/release-$libevent_version/libevent-$libevent_version.tar.gz.asc" -o libevent-$libevent_version.tar.gz.asc && \
-  gpg --import /libevent.gpg && \
-  gpg libevent-$libevent_version.tar.gz.asc && \
-  shasum -a 256 libevent-$libevent_version.tar.gz && \
+  gpg --keyring /libevent.gpg --verify libevent-$libevent_version.tar.gz.asc libevent-$libevent_version.tar.gz && \
   echo "$libevent_hash  libevent-$libevent_version.tar.gz" | shasum -a 256 -c - && \
   tar -zxvf libevent-$libevent_version.tar.gz && \
   cd libevent-$libevent_version && \
   ./configure \
-              LDFLAGS="-L$PWD/../openssl-$openssl_version/root" \
-              CPPFLAGS="-I$PWD/../oppenssl-$openssl_version/include" \
+              --disable-openssl \
```

Comment:
i couldn't get windows/linux to build libevent with openssl support, but it seems ok to disable based on the examples in https://gitlab.torproject.org/tpo/core/tor/-/issues/40111 and https://github.com/ahf/tor-win32/blob/master/Makefile#L66. definitely needs testing though.

Reply:
did you try adding `--enable-static-libevent` when configuring tor? it will point to the correct static library location in libevent dir
Also what is the error you got?

Reply:
nvm, this seems to be building libevent only error